dd-trace-api/src/main/java/datadog/trace/api/Config.java

```diff
@@ -75,6 +75,15 @@
   public static final String LANGUAGE_TAG_VALUE = "jvm";
   public static final String DEFAULT_SERVICE_NAME = "unnamed-java-app";
 
+  public static final String TRACING_LIBRARY_KEY = "signalfx.tracing.library";
+  public static final String TRACING_LIBRARY_VALUE = "java-tracing";
+  public static final String TRACING_VERSION_KEY = "signalfx.tracing.version";
+  public static final String TRACING_VERSION_VALUE = "0.24.0-sfx9";
```

Comment:
We need to document our release process and make sure this gets updated properly when making a release.